waspc/src/Wasp/Project/ExternalConfig.hs

```diff
@@ -23,11 +23,12 @@ data ExternalConfigs = ExternalConfigs
   deriving (Show)
 
 analyzeExternalConfigs ::
+  Bool ->
```

Comment:
In general, seeing a Bool like this, where I can't tell from signature at all what it could be, is a bit suspicious. I commented more above why this one is probably not the right choice here.